modules/gltf/src/lib/extensions/deprecated/EXT_feature_metadata.ts

```diff
@@ -127,24 +127,73 @@ function getPropertyDataFromBinarySource(
   schemaProperty: GLTF_EXT_feature_metadata_ClassProperty,
   numberOfFeatures: number,
   featureTableProperty: GLTF_EXT_feature_metadata_FeatureTableProperty
-): Uint8Array | string[] {
+): BigTypedArray | string[] {
   const bufferView = featureTableProperty.bufferView;
   const dataArray: Uint8Array = scenegraph.getTypedArrayForBufferView(bufferView);
 
-  switch (schemaProperty.type) {
-    case 'STRING': {
-      const offsetsData = getStringOffsets(scenegraph, featureTableProperty, numberOfFeatures);
-      if (!offsetsData) {
-        return [];
-      }
-      return getStringAttributes(dataArray, offsetsData, numberOfFeatures);
+  if (schemaProperty.type === 'STRING') {
+    const offsetsData = getStringOffsets(scenegraph, featureTableProperty, numberOfFeatures);
+    if (!offsetsData) {
+      return [];
     }
-    default:
+    return getStringAttributes(dataArray, offsetsData, numberOfFeatures);
+  } else if (isNumericProperty(schemaProperty.type)) {
+    return getNumericAttributes(
+      dataArray,
+      schemaProperty.type as
+        | 'INT8'
+        | 'UINT8'
+        | 'INT16'
+        | 'UINT16'
+        | 'INT32'
+        | 'UINT32'
+        | 'INT64'
+        | 'UINT64'
+        | 'FLOAT32'
+        | 'FLOAT64',
+      numberOfFeatures
+    );
   }
 
   return dataArray;
 }
 
+/**
+ * Check if the feature table property is of numeric type
+ * @param schemaPropertyType - feature table property
+ * @returns true if property is numeric, else - false
+ */
+function isNumericProperty(
+  schemaPropertyType:
+    | 'INT8'
+    | 'UINT8'
+    | 'INT16'
+    | 'UINT16'
+    | 'INT32'
+    | 'UINT32'
+    | 'INT64'
+    | 'UINT64'
+    | 'FLOAT32'
+    | 'FLOAT64'
+    | 'BOOLEAN'
+    | 'STRING'
+    | 'ENUM'
+    | 'ARRAY'
+    | string
```

Comment:
By adding `string` here, all the other constants become meaningless. The resulting type is `string`

Reply:
It is to comply with the spec